Add more info about install and uploading

Guseyn opened the pull request:
resolve #9

g4s8 commented:
@0crat assign me

0crat commented:
I'm not managing `artipie/pypi-adapter` GitHub repository, you have to contact me in Slack first, see [§11](http://www.zerocracy.com/policy.html#11)

g4s8 commented:
@0crat in

Guseyn commented:
@g4s8 @yegor256 could please someone start proceeding this pr?

amihaiemil commented:
@g4s8 Assign me, I have time now :)

Guseyn commented:
@amihaiemil I assigned you

g4s8 commented:
@0crat assign @amihaiemil

0crat commented:
> @0crat assign @amihaiemil  ([here](https://github.com/artipie/pypi-adapter/issues/5#issuecomment-581870174))

@g4s8 There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 11@377a78bde909, thread: PQ-CT2E6TK9B
com.zerocracy.claims.ClaimIn[214] java.lang.IllegalArgumentException: Parameter "age" not found in "Order was canceled" among: [comment, job, issue, repo, cause, _parent_sig, flow, login]
```

[0.51.0](https://github.com/zerocracy/farm): CID: [518ce9bd-f4aa-4cd4-a457-40ec9c036f2c](https://www.0crat.com/footprint/CT2E6TK9B/518ce9bd-f4aa-4cd4-a457-40ec9c036f2c), Type: "Order was canceled"

<!-- https://www.0crat.com/footprint/CT2E6TK9B/9a188763-f80e-4cc0-b8b9-549dccc22b9c, version: 0.51.0, hash: ${buildNumber} -->

0crat commented:
> @0crat assign @amihaiemil  ([here](https://github.com/artipie/pypi-adapter/issues/5#issuecomment-581870174))

@g4s8 This pull request [#5](https://github.com/artipie/pypi-adapter/issues/5) is assigned to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/4626fb00-1547-4147-a00d-c20d9340089f); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

<!-- https://www.0crat.com/footprint/CT2E6TK9B/6492ffbc-ccd3-4320-8db1-cbade777ebeb, version: 0.51.0, hash: ${buildNumber} -->

0crat commented:
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

<!-- https://www.0crat.com/footprint/CT2E6TK9B/f1f49fbc-108e-4c32-b491-015b09535812, version: 0.51.0, hash: ${buildNumber} -->

amihaiemil commented on README.md:
````diff
@@ -41,37 +41,59 @@ $ mvn clean install -Pqulice
+
+```python
+pip install -i http://localhost:8080/ --upgrade example-pkg-YOUR-USERNAME-HERE
+```
````
@Guseyn Why did you add the below empty files? ``/example_pkg/README.md`` and ``/example_pkg/__init__.py``? Maybe you can write some dummy texts inside of them so their purpose becomes clearer?

Guseyn replied:
@amihaiemil because it's sample project for future tests, probably I will remove it later. I just need it right now.

amihaiemil commented on README.md:
```diff
@@ -41,37 +41,59 @@ $ mvn clean install -Pqulice
-For uploading using pip one must create a valid ~/.pypirc file:
+Follow these instructions: [https://packaging.python.org/tutorials/packaging-projects/](https://packaging.python.org/tutorials/packaging-projects/)
+
+1. For uploading using pip one must create a valid ~/.pypirc file:
```
@Guseyn Can you make this a Header using ``###`` (it will be a Header with a lower case than the parent ``How it works``). 

And can you make it simpler: just ``Uploading using pip`` is enough. The statement ``You must create a valid ~/.pypric file`` can be the first sentence under the header.

Guseyn replied:
I will structure the information in a better way later, it's just a reminder to me what kinda instructions that I need.

amihaiemil commented on README.md:
````diff
@@ -41,37 +41,59 @@ $ mvn clean install -Pqulice
-You can find sample files in the [test resources](https://github.com/artipie/pypi-adapter/tree/master/src/test/resources/simple-pypi-project)
+```
+
+2. For installing run the command
````
@Guseyn This should also be a Header (``##``) and I think it should be: ``Installing a Package using pip``

Guseyn replied:
I will structure the information in a better way later, it's just a reminder to me what kinda instructions that I need.

Guseyn replied:
@amihaiemil right now it just information, I will make readme document more elegant later.

amihaiemil commented on README.md:
```diff
@@ -41,37 +41,59 @@ $ mvn clean install -Pqulice
 [distutils]
-index-servers = example
+index-servers=
+    testpypi
```
@Guseyn The indentation here seems strange. Why not in one line? ``index-servers = testpypi`` ?

Guseyn replied:
because it's the format that I found in docs.

amihaiemil commented on README.md:
```diff
@@ -41,37 +41,59 @@ $ mvn clean install -Pqulice
-username = myname
-password = mypass
+[testpypi]
+repository: http://localhost:8080/
```
@Guseyn Is the ``repository`` keyword needed for PiP? If it can be anything, I would name it ``url``.

Guseyn replied:
@amihaiemil  Yes, it's configuration property, so it must be 'repository'

amihaiemil commented on README.md:
````diff
@@ -41,37 +41,59 @@ $ mvn clean install -Pqulice
 ```
 
-For using a private server with setuptools unittesting you need to add the following to your setup.py:
+Where `setup.py` looks smth like:
````
@Guseyn Please don't use Internet abbreviations in READMEs. ``something like this`` instead of ``smth like``. 

Also, can you give a little more context to these examples? For instance, in the ``uploading`` section, I understand you have to run your own repository server, but the ``setup.py`` could use more details: what project are we actually uploading in this example?

Guseyn replied:
@amihaiemil  I will fix all stuff later, right now I need such information for further development.

amihaiemil commented:
@Guseyn Also, please link the Github Issue that this PR is solving and provide a decent description of this PR. Something like.

Guseyn commented:
@amihaiemil plz approve the changes

amihaiemil commented:
@Guseyn Why "fix them later"? If you wanted to fix stuff later, why is Code Review necessary then? 

Please fix them now. At least the headers stuff and the abbreviations...

amihaiemil commented:
@Guseyn Again, please don't imagine you are the only developer here. Maybe it's just you at the moment, but this project will have more DEVs probably. So don't do stuff just for yourself like "leaving reminders"...

Guseyn commented:
@amihaiemil  "please don't imagine you are the only developer here" => I don't imagine. Because I am the only developer here, and I explained that the information that I added is row, and decided format it and structure it in other pr. 
"why is Code Review necessary then?" this question is not for me. I just added info in readme file, but never mind. I close pr.

0crat commented:
Order was finished: +20 point(s) just awarded to @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

<!-- https://www.0crat.com/footprint/CT2E6TK9B/c7dc2bcd-5e74-4168-9152-7d08e86c8123, version: 0.51.0, hash: ${buildNumber} -->

0crat commented:
Job was finished in 1 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

<!-- https://www.0crat.com/footprint/CT2E6TK9B/095b303e-66cd-412c-b0bb-cedd6a5e0614, version: 0.51.0, hash: ${buildNumber} -->

0crat commented:
This pull request [#5](https://github.com/artipie/pypi-adapter/issues/5) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/2b2a3a42-81e1-48fb-9b88-a6561a16b018); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

<!-- https://www.0crat.com/footprint/CT2E6TK9B/82ce99f9-e21f-444d-827d-72f91aa41d1e, version: 0.51.0, hash: ${buildNumber} -->

0crat commented:
Pull request [#5](https://github.com/artipie/pypi-adapter/issues/5) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

<!-- https://www.0crat.com/footprint/CT2E6TK9B/04ef4ccf-f6b0-4e55-a4f2-1310c9789253, version: 0.51.1, hash: ${buildNumber} -->

0crat commented:
Job [#5](https://github.com/artipie/pypi-adapter/issues/5) is already in the agenda of @amihaiemil[/z](https://www.0crat.com/u/amihaiemil)

<!-- https://www.0crat.com/footprint/CT2E6TK9B/6969ee48-1f1d-46ec-9d27-b44768e35cc2, version: 0.51.1, hash: ${buildNumber} -->

g4s8 commented:
@0crat refuse

g4s8 commented:
@0crat out

0crat commented:
> @0crat out ([here](https://github.com/artipie/pypi-adapter/issues/5#issuecomment-582269489))

@g4s8 Job `gh:artipie/pypi-adapter#5` is not assigned, can't get performer

<!-- https://www.0crat.com/footprint/CT2E6TK9B/ee0fe493-d7e4-41bb-b66c-b19f2ba88f5b, version: 0.51.1, hash: ${buildNumber} -->

0crat commented:
> @0crat refuse ([here](https://github.com/artipie/pypi-adapter/issues/5#issuecomment-582269465))

@g4s8 Job `gh:artipie/pypi-adapter#5` is not assigned, can't get performer

<!-- https://www.0crat.com/footprint/CT2E6TK9B/ae34bf46-53a2-452d-b7c1-96b9ed830f81, version: 0.51.1, hash: ${buildNumber} -->

0crat commented:
This job is not in scope

<!-- https://www.0crat.com/footprint/CT2E6TK9B/c3d74e4e-e05c-4ed5-b6cd-57ecb8857db7, version: 0.51.1, hash: ${buildNumber} -->
